### ISSUE TITLE
How can I make the indent guide continuous even on line with white space? 

### ISSUE THREAD

**Issue by flashery** [DarthFennec/highlight-indent-guides#31]
Hello I love you plugin and I am currently using it now on my Emacs. One question though. 
How can I make the indent guide continuous even on line with white space?  Like sublime text indent guide. Also, what indent guide character should I use to make it look like sublime text indent guide?

**Comment by DarthFennec** [comment]
I presume by "line with white space" you mean empty lines with nothing in them? You're not the first person to ask for this feature (see #19 and #21), and I've heavily considered adding it. But I don't think I'm going to unless I find a good way to implement it (which I think is unlikely to happen). Every approach I've found so far would impact either the performance or the complexity/stability of the mode to a degree that I don't think is worthy of this feature. In Sublime it's easy because the indent highlighting is integrated into the editor, but I have to work within the restrictions of Emacs, and sometimes those things just don't work out.

Of course I'm willing to be proven wrong, and I'm still actively looking for a way to do this that doesn't complicate the code too much, doesn't impact performance too much, and doesn't create too many incompatibilities with other modes. But so far it seems unlikely that this will happen.

As for your second question, it looks like sublime uses a dotted line? There are a number of characters you can use for a dotted or dashed line, although none of them really look exactly like Sublime's version. Here's a few: `¦` (U+00A6), `⁞` (U+205E), `⋮` (U+22EE), `┆` (U+2506), `┇` (U+2507), `┊` (U+250A), `┋` (U+250B), `╎` (U+254E), `╏` (U+254F), `⸽` (U+2E3D), `︙` (U+FE19). Of course, which of these look the best depends on your fonts etc, so you can just try a few that you think you might like and see what works. I've played with the idea of allowing an arbitrary bitmap instead of a character, so people can make it look like anything they want, but for now these are probably the best options.

**Comment by saf-dmitry** [comment]
> I've played with the idea of allowing an arbitrary bitmap instead of a character

I would like the idea of using bitmaps. This way we could probably account for different line height settings. Did you consider creating an XPM image from generated bitmap data?

**Comment by DarthFennec** [comment]
@saf-dmitry Yeah, I was thinking I'd generate the XPM programmatically based on the current line height (and whatever other measurements/options might be available). Like how `fci-mode` works.

**Comment by DarthFennec** [comment]
#35 is done, let me know if you have any problems with it.
